api/api/accounts/test_list_userroles.py

```diff
@@ -60,8 +58,13 @@ def testListRoles_userIsCoreMember_returnsFirstFiftyRolesList(client, test_user,
                             assigned_at=date.today(), active=True)
 
     # when
-    response = client.get(f'/users/{test_user.id}/roles/', format='json')
+    response = client.get('/users/accounts/me/roles/', format='json')
 
     # then
     assert response.status_code == status.HTTP_200_OK
     assert len(response.data['results']) == 50
+    assert response.data['results'][0] == {'name': 'Coordinator',
+                                           'club__name': 'Bitbyte - The Programming Club',
+                                           'club__slug': 'slug field',
+                                           'assigned_at': '2022-08-31',
```

Comment:
wouldn't it fail on 1st September 2022?

Reply:
Why ?

Reply:
While creating the role, you're setting `assigned_at`  by using date.today(), which will be different on different days. But you're still asserting it with 31st August 2022.